mozci/push.py

```diff
@@ -1203,14 +1213,17 @@ def classify_regressions(
         groups_non_consistent_failure = {
             g.name
             for g in all_groups
-            if g.status != Status.PASS
-            and g.is_consistent_failure(
-                cross_config_counts[0] if cross_config_counts is not None else None,
-                consistent_failures_counts[0]
-                if consistent_failures_counts is not None
-                else None,
+            if groups_confirmed[g.name] is False
+            or (
+                g.status != Status.PASS
+                and g.is_consistent_failure(
+                    cross_config_counts[0] if cross_config_counts is not None else None,
+                    consistent_failures_counts[0]
+                    if consistent_failures_counts is not None
+                    else None,
+                )
+                is False
             )
-            is False
         }
```

Comment:
Similarly here, with this approach, if a manifest is not failing multiple times but the corresponding -cf tasks are failing (so g.is_confirmed is True and g.is_consistent_failure is False) we still consider a group as an intermittent failure. Should we instead do something like:
```g.is_confirmed is False or (g.is_confirmed is None and g.is_consistent_failure is False)```?

Reply:
good catch, otherwise here we would end up allowing `g.is_confirmed and !g.is_consistent_failure`